src/app/shared/components/custom-button/custom-button.component.html

```diff
@@ -0,0 +1,3 @@
+<button [ngClass]="defaultClasses + ' ' + additionalClasses">
```

Comment:
if you sure that custom-button class will be always here you can left it in html and remove variable in ts file

Reply:
```
<button class="custom-button" [ngClass]="additionalClasses">
    <ng-content></ng-content>
</button>
```
Fixed!